### ISSUE TITLE
Gutenboarding: make loading state accessible; describe what's happening

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#43553]
At `/new` onboarding flow.

> The final step before diving into Gutenberg, with the animation showing “Building your site” was not accessible – nothing is read by VoiceOver here

![image](https://user-images.githubusercontent.com/87168/85380473-a68f4700-b545-11ea-9933-3d2d31b3a164.png)

Likely `aria-live="polite" aria-busy="true"` needed?

**Comment by youbrokesomething** [comment]
The onboarding flow has changed many times since this ticket was opened. I'm going to close this in favor of new audit findings.
